### PR TITLE
Tutorial like hexdocs

### PR DESCRIPTION
**Pull request by chouzar**
Added a handful of additions to the docs:

* A mention to ayesql as an alternative elixir library
* Setup README.md as the main entry point for docs :)
* Added `postgrex` and `ecto` configuration guides

## Preview  

guides

---

This pull request is meant to solve #6 and #3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,19 @@ eql is an Erlang library with similar inspiration and goals.
...
 
+### [ayesql](https://github.com/alexdesousa/ayesql)
+
+ayesql is another Elixir library, a bit more powerful than yesql:
+
+- It offers support for various SQL statements in a single file.
+- Special constructs for [query composability](https://hexdocs.pm/ayesql/readme.html#query-composition) in SQL files.
+- Special constructs for [optional parameters](https://hexdocs.pm/ayesql/readme.html#optional-parameters) in SQL files.
+
+Should I use ayesql then? Not necessarily... it may depend on your style preference.
```
Comment (chouzar) [inline README.md]: Unsure about the wording here, it may not even be a "style preference".

File: README.md [hunk 2]
```diff
@@ -87,6 +87,19 @@ eql is an Erlang library with similar inspiration and goals.
...
+
+yesql will keep your SQL queries closer to canonical SQL, but if you start finding
+it limiting or convoluted maybe it would be a good time to check out more powerful
+abstractions like ayesql or Ecto.
```
Comment (chouzar) [inline README.md:100]: Here I'm basically repeating what is mentioned above "move to ecto if you have the need".

**Comment by chouzar** [comment]
Sorry for push-forcing just re-arranged commits so they had a logical follow through 😅

File: README.md (hunk 1 above)
Comment (lpil) [reply README.md:98]: I think we can remove this line andusrs can decide for themselves

File: README.md [hunk 3]
```diff
@@ -95,8 +95,6 @@ ayesql is another Elixir library, a bit more powerful than yesql:
 - Special constructs for [query composability](https://hexdocs.pm/ayesql/readme.html#query-composition) in SQL files.
 - Special constructs for [optional parameters](https://hexdocs.pm/ayesql/readme.html#optional-parameters) in SQL files.
 
-Should I use ayesql then? Not necessarily... it may depend on your style preference.
-
```
Comment (chouzar) [inline README.md]: Removed this sentence, but feel free to point out if anything feels off with the wording or examples.

**Review by lpil — APPROVED** [review]
Thank you

**Pull request merged**